sources/Outputs/src/Dyd.cpp

```diff
@@ -269,6 +295,11 @@ Dyd::writeGenConnect(const algo::GeneratorDefinition& def, unsigned int index) {
   return {connection, signal};
 }
 
+void
+Dyd::writeVRRemoteConnect(const boost::shared_ptr<dynamicdata::DynamicModelsCollection>& collection, const std::string& busId) {
```

Comment:
Should be done via a builder to respect previous developements

Reply:
Could you develop what you mean by the "builder" ?

Reply:
The global idea for these functions "write..." is that they generate an item and only that. That way, the function implements the builder pattern and then we can manipulate the item. But with your function, you build the item AND you add it to the collection AND you perform the connections

Reply:
But for this function "writeVRRemoteConnect" if I don't pass the collection to the function then I will be unable to add the connect, I would have to do this in the write function.

Reply:
OK for the connections. My problem was more about the writeGenerator function